src/emmc.c

```diff
@@ -12,6 +12,8 @@
 #include "utils.h"
 #include "emmc.h"
 
+#define SECONDARY_GPT_SECTOR_COUNT 34
```

Comment:
Can we rename this to `PARTITION_TABLE_SIZE_GPT`? :slightly_smiling_face: This way we can have different partition table sizes for `gpt` and `msdos`. I am working on a related bug fix which uses these:
```c
#define PARTITION_TABLE_SIZE_MSDOS 1
#define PARTITION_TABLE_SIZE_GPT   34
```
"Size" sounds more fitting than "count", as this is a contiguous block of memory on disk and not something like an index or offset.

A different question that crossed my mind: Is the secondary partition table for GPTs of same size as the primary? If not we have do define them differently, of course:
```c
#define PARTITION_TABLE_SIZE_MSDOS           1
#define PARTITION_TABLE_SIZE_GPT_PRIMARY     42
#define PARTITION_TABLE_SIZE_GPT_SECONDARY   34
```

Reply:
Sure, I will rename this here.
Both GPTs are the same size as the secondary GPT is a backup of the primary GPT.